### ISSUE TITLE
Can it be used in nodejs ?

### ISSUE THREAD

**Issue by elyotechgit** [abdmob/x2js#17]
Hi, 

It's a very great tool, 

I'm looking to use in node server, is it possible? for now I'm facing problem with window object error.

Could you help me with that ?

thanks a lot

**Comment by abdolence** [comment]
I think it's possible and I'll check this in the next version. Thanks!

**Comment by elyotechgit** [comment]
I already tried a bit, but I face problem with the DomParser of the window, as it's not an window.

and the regular domparser is not good enough to parse complex XML.

can we discuss that (yonia@elyotech.com)

**Comment by abdolence** [comment]
Yes, it looks like there is no "standard" DOM parser for node and it's required to use external (
https://github.com/nodejs/node-v0.x-archive/wiki/modules#parsers-xml ). 
Actually I'm not using nodejs for myself - do you know any popular/suitable?

**Comment by abdmob** [comment]
What about node-expat?

**Comment by elyotechgit** [comment]
Didn't tried it, but I will , thanks

_Yoni Alhadeff_
Elyotech LTD
IT Team
Tel: +972 (0) 72 222 33 02

Mobile: +972 (0) 52 873 96 96
laurentl@elyotech.comyonia@elyotech.com

http://www.elyotech.com

**Comment by AuSong** [comment]
this issue still opening?
